client/state/atomic/software/reducer.js

```diff
@@ -0,0 +1,15 @@
+import { withStorageKey } from '@automattic/state-utils';
+import { ATOMIC_PLUGIN_INSTALL_SET_STATUS } from 'calypso/state/action-types';
+import { keyedReducer } from 'calypso/state/utils';
+
+function software( state = {}, action ) {
+	switch ( action.type ) {
+		case ATOMIC_PLUGIN_INSTALL_SET_STATUS:
+			return { ...state, softwareSet: action.softwareSet, ...action.status };
+	}
+}
+
+export default withStorageKey(
+	'atomicSoftware',
+	keyedReducer( 'siteId', keyedReducer( 'softwareSet', software ) )
+);
```

Comment:
I don't really understand these but I think I got it working in the right order. Keen to understand why we need a withStorageKey and registerReducer with the same key.